docs/docs-ref-autogen/excel/excelscript/excelscript.searchcriteria.yml

```
@@ -4,7 +4,34 @@ uid: 'ExcelScript!ExcelScript.SearchCriteria:interface'
 package: ExcelScript!
 fullName: ExcelScript.SearchCriteria
 summary: Represents the search criteria to be used.
-remarks: ''
+remarks: |-
+
+
+  #### Examples
+
+  ```TypeScript
+  /**
+   * This script looks for the next instance of the text "TK" on the current worksheet.
+   * It then selects that cell and removes "TK" and all formatting from the cell.
+   */
+  function main(workbook: ExcelScript.Workbook) {
+    // Get the used range on the current worksheet.
+    let range = workbook.getActiveWorksheet().getUsedRange();
+
+    // Get the next cell that contains "TK".
+    let tkCell = range.find("TK", {
+      completeMatch: true, /* Don't match if the cell text only contains "TK" as part of another string. */
+      matchCase: false,
+      searchDirection: ExcelScript.SearchDirection.forward /* Start at the beginning of the range and go to later columns and rows. */
+    });
+
```

```suggestion
if (tkCell == undefined) return;
```
Check for when text not found to avoid an error on next lines of code.